src/app/components/FeelsLikeCard.tsx

```diff
@@ -0,0 +1,36 @@
+import { roundNumber } from "../util/math";
+import WeatherCard from "./WeatherCard";
+import { WiThermometer } from "rocketicons/wi";
+
+interface FeelsLikeCardProps {
```

Comment:
```suggestion
export interface FeelsLikeCardProps {
```
export props interface